### ISSUE TITLE
UI improvement - more consistency

### ISSUE THREAD

**Issue by frederikb96** [scubajeff/lespas#131]
I am using Les Pas together with friends and family and one thing I realized was that they found it difficult at some points to understand the GUI easily. So I would like to give a recommendation on this basis, but feel free to simply neglect this if you think otherwise :)

One little problem was the different design choice for the same functionality on different windows. I can remember on two things right now: 

1. The move/coppy button is different if you are in the gallery or in an Les Pas album and have pictures selected as presented on the screenshots here:

Proposed solution: Use the same three icons (Trash, Share and Les Pas icon) on both windows if a picture is selected. So basically replace the "to album" in this submenu by a les pas icon. 

2. Refering to #126: When creating a new album, a magnefying glas is displaced to filter by name, but on the main screen it is only available via the filter by name entry in the menu.

Proposed solution: On the main window, I would directly display the magnifying glas and ones it is clicked, there can be this light gray text showing filter by name. This way, the search functionality can still be called search in the menu, which is used to search for faces and so on. So no renaming necessary, but would be more intuitive for normal users.

**Comment by scubajeff** [comment]
Give me sometime to digest.

One quick tips, if ever wonder what an icon will do, just long press it and tooltips will be shown.

**Comment by scubajeff** [comment]
I agree consistency is good UI design practice. Menu items like Share, Delete and Add it to LesPas, could be in the same order in multiple screens. My only argument here is we also need to consider putting the most frequently used ones up in the list, this is important because screen width is not the same of every phones, there could only be two or even one slot left for function icons rather than three as in your screenshots. When user is in album photo list view, the most frequently used function got to be Delete and Sharing out. But in Gallery screen, 'Adding it to Les Pas' is somehow more important than the other two.

Regarding the search function, I'm considering providing a consolidating search experience backed by a simple UI. Until then, I'm afraid there will be no enhancement to the current ones.

**Comment by scubajeff** [comment]
I should warn you, new search UI is not gonna be here any time soon. ;)
